app/tests/test_search.py

```diff
@@ -1,19 +1,17 @@
-from unittest.mock import patch
-
 from bs4 import BeautifulSoup
 from flask.testing import FlaskClient
-from sqlalchemy import exc
 
-from app.main.db.queries import fuzzy_search
-from app.tests.mock_database import create_two_test_files
+from app.tests.mock_database import create_multiple_test_records
 
 
-def test_search_get(client: FlaskClient):
+def test_search_get(client: FlaskClient, app):
     """
     Given a user accessing the search page
     When they make a GET request
     Then they should see the search form and page content.
     """
+    app.config["DEFAULT_PAGE_SIZE"] = 5
```

Comment:
since most of these tests dont care about DEFAULT_PAGE_SIZE, just set it in testing_config.py by overriding

```
class TestingConfig(BaseConfig):
    TESTING = True
    SECRET_KEY = "TEST_SECRET_KEY"  # pragma: allowlist secret
    WTF_CSRF_ENABLED = False
    DEFAULT_PAGE_SIZE = 5

    def _get_config_value(self, variable_name):
        return ""

```

you can leave explicitly overriding it in tests which care about the size of the page

Reply:
done the changes as requested